[Bug]: TG适配器代理配置不生效！！！

### 版本信息

v2.0.7

### 部署方式

Docker 一键部署（含 Napcat）

### 运行环境

Linux

### 系统架构

amd64 (x86_64)

### Python 版本

_No response_

### 发生了什么？

配置TG适配器填写Token等
重启NekroAgent
提示
```shell
10-26 23:42:21 [INFO] nekro_agent | init:115| Telegram 适配器使用代理: http://10.31.1.242:7890
10-26 23:42:26 [ERROR] nekro_agent | init:144| Telegram 适配器初始化失败: Timed out
10-26 23:42:26 [ERROR] nekro_agent | cleanup:206| Telegram 适配器清理失败: This Updater is not running!
```
我在容器内使用了curl测试
```shell
# curl -x http://10.31.1.242:7890 -v "https://api.telegram.org/bot<我的token>/getMe"
*   Trying 10.31.1.242:7890...
* Connected to 10.31.1.242 (10.31.1.242) port 7890 (#0)
* allocate connect buffer!
* Establish HTTP proxy tunnel to api.telegram.org:443
> CONNECT api.telegram.org:443 HTTP/1.1
> Host: api.telegram.org:443
> User-Agent: curl/7.74.0
> Proxy-Connection: Keep-Alive
> 
< HTTP/1.1 200 Connection established
< 
* Proxy replied 200 to CONNECT request
* CONNECT phase completed!
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*  CAfile: /etc/ssl/certs/ca-certificates.crt
*  CApath: /etc/ssl/certs
* TLSv1.3 (OUT), TLS handshake, Client hello (1):
* CONNECT phase completed!
* CONNECT phase completed!
* TLSv1.3 (IN), TLS handshake, Server hello (2):
* TLSv1.3 (IN), TLS handshake, Encrypted Extensions (8):
* TLSv1.3 (IN), TLS handshake, Certificate (11):
* TLSv1.3 (IN), TLS handshake, CERT verify (15):
* TLSv1.3 (IN), TLS handshake, Finished (20):
* TLSv1.3 (OUT), TLS change cipher, Change cipher spec (1):
* TLSv1.3 (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / TLS_AES_256_GCM_SHA384
* ALPN, server accepted to use h2
* Server certificate:
*  subject: CN=api.telegram.org
*  start date: Mar 25 13:09:41 2025 GMT
*  expire date: Apr 26 13:09:41 2026 GMT
*  subjectAltName: host "api.telegram.org" matched cert's "api.telegram.org"
*  issuer: C=US; ST=Arizona; L=Scottsdale; O=GoDaddy.com, Inc.; OU=http://certs.godaddy.com/repository/; CN=Go Daddy Secure Certificate Authority - G2
*  SSL certificate verify ok.
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Using Stream ID: 1 (easy handle 0x55c3d4371320)
> GET /bot8331984307:AAGG995EsSV1S_iP6kns6g9sNJsWt7m6Ups/getMe HTTP/2
> Host: api.telegram.org
> user-agent: curl/7.74.0
> accept: */*
> 
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* old SSL session ID is stale, removing
* Connection state changed (MAX_CONCURRENT_STREAMS == 128)!
< HTTP/2 200 
< server: nginx/1.18.0
< date: Sun, 26 Oct 2025 15:53:08 GMT
< content-type: application/json
< content-length: 261
< strict-transport-security: max-age=31536000; includeSubDomains; preload
< access-control-allow-origin: *
< access-control-allow-methods: GET, POST, OPTIONS
< access-control-expose-headers: Content-Length,Content-Type,Date,Server,Connection
< 
* Connection #0 to host 10.31.1.242 left intact
{"ok":true,"result":{"id":返回的ID,"is_bot":true,"first_name":"\u96ea\u98cenanoda","username":"NAME","can_join_groups":false,"can_read_all_group_messages":true,"supports_inline_queries":false,"can_connect_to_business":false,"has_main_web_app":false}}
```
(部分涉及安全部分已抹除)

### 相关信息

```shell

```

### 贡献意向

可以提供更详细的问题重现步骤

### 补充信息

_No response_

--- a/nekro_agent/adapters/telegram/adapter.py
+++ b/nekro_agent/adapters/telegram/adapter.py
@@ -113,11 +113,15 @@ class TelegramAdapter(BaseAdapter[TelegramConfig]):
             proxy_url = self.config.PROXY_URL.strip() if self.config.PROXY_URL else None
             if proxy_url:
                 logger.info(f"Telegram 适配器使用代理: {proxy_url}")
-            
-            # 初始化 Application
-            self.application = (
-                Application.builder().token(self.config.BOT_TOKEN).build()
-            )
+
+            # 初始化 Application，配置代理
+            builder = Application.builder().token(self.config.BOT_TOKEN)
+            if proxy_url:
+                # 配置代理：同时为常规API请求和轮询配置代理
+                # proxy: 用于常规Bot API请求（发送消息、下载文件等）
+                # get_updates_proxy: 用于轮询获取更新
+                builder = builder.proxy(proxy_url).get_updates_proxy(proxy_url)
+            self.application = builder.build()
 
             # 初始化消息处理器
             self.message_processor = MessageProcessor(self)

